app/controllers/api/v1/CommentsController.js

```diff
@@ -48,6 +48,12 @@ export default class CommentsController {
 
       const timelines = await newComment.create()
 
+      timelines.forEach(async (timeline) => {
+        if (timeline.isDirects()) {
+          await PubSub.updateUnreadDirects(timeline.userId)
+        }
+      })
```

Comment:
forEach doesn't know how to handle asyncs, so you're ignoring errors coming from the callback. Use `await Promise.all(timelines.map(…))` instead